frontend/src/components/landing-page/mock-new-cars.ts

```diff
@@ -0,0 +1,129 @@
+declare type ModelResponseDto = {
+  id: string;
+  wishlistId: string | null;
+  createdAt: Date;
+  name: string;
+  yearStart: number;
+  yearEnd: number | null;
+  photoUrls: string[] | [];
+  brand: {
+    name: string;
+    logoUrl: string;
+  };
+  bodyType: string;
+  manufactureCountry: string;
+  pricesRanges: {
+    price_start: number;
+    price_end: number;
+  }[];
+  description: string;
+  complectationName?: string;
+};
```

Comment:
Don't we already have a type for the model response?
@iryb, could you help me and @setar92 find that if you faced it?

Reply:
@oleksandr-danylchenko @setar92 

Yes, it is in shared folder.

https://github.com/BinaryStudioAcademy/bsa-2022-autoline/blob/develop/shared/src/common/types/preferences/wishlist-response-dto.type.ts